scripts/item-descriptions.lua

```diff
@@ -0,0 +1,75 @@
+-- Holds custom descriptions for view-item-info
+-- By PeridexisErrant
+
+print("This is a content library; calling it does nothing.")
```

Comment:
Oh, your idea is that this script can be overridden on a per-save basis. I'd recommend implementing `desc_of_item` in a Lua library and loading item descriptions from other Lua files/scripts, which could simply contain a table of item descriptions:
```lua
{ANVIL =  {"An essential component of the forge."}}
```